ompi/mpi/fortran/mpif-h/Makefile.am

```diff
@@ -111,7 +111,7 @@ CLEANFILES += sizeof_f.f90
 # out this workaround indefinitely.  So Jeff advised Absoft to fix
 # this bug in Sep 2014.
 noinst_LTLIBRARIES=
-if BUILD_FORTRAN_SIZEOF
+if BUILD_PMPI_FORTRAN_MPIFH_BINDINGS_LAYER
```

Comment:
Since we want to get v1.10 out ASAP, I might take a whack at this and do a new PR today.  I'll see what I can do.